lib/doorkeeper/orm/active_record.rb

```diff
@@ -12,6 +12,27 @@ def self.initialize_models!
           end
         end
       end
+
+      def self.initialize_application_owner!
```

Comment:
It refer to #281 .
`initialize_application_owner!` implemented in ORM Adapter is good to override, or can only extend by monkey patch